sql/core/benchmarks/InExpressionBenchmark-results.txt

```diff
@@ -0,0 +1,544 @@
+Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz
+5 shorts:                                Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------
+In expression                                   29 /   32        339.2           2.9       1.0X
+InSet expression                               110 /  115         90.5          11.0       0.3X
+
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_192-b12 on Mac OS X 10.14
+Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz
+10 shorts:                               Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------
+In expression                                   36 /   42        280.1           3.6       1.0X
+InSet expression                               121 /  130         83.0          12.1       0.3X
+
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_192-b12 on Mac OS X 10.14
+Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz
+25 shorts:                               Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------
+In expression                                   50 /   52        199.1           5.0       1.0X
+InSet expression                               125 /  128         80.0          12.5       0.4X
+
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_192-b12 on Mac OS X 10.14
+Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz
+50 shorts:                               Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------
+In expression                                   81 /   84        123.8           8.1       1.0X
+InSet expression                               159 /  164         62.7          15.9       0.5X
+
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_192-b12 on Mac OS X 10.14
+Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz
+100 shorts:                              Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------
+In expression                                  142 /  144         70.5          14.2       1.0X
+InSet expression                               135 /  140         74.1          13.5       1.1X
+
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_192-b12 on Mac OS X 10.14
+Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz
+200 shorts:                              Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------
+In expression                                  259 /  265         38.6          25.9       1.0X
+InSet expression                               134 /  139         74.5          13.4       1.9X
```

Comment:
It's good to compare `In` and `InSet`, but it seems to be difficult to see the trend. Let me think about this.